src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -36,7 +36,7 @@ func AddKafkaFlags(
 		brokers,
 		"brokers",
 		[]string{},
-		"Comma-separated list of broker ip:port pairs (e.g."+
+		"Comma-separated list of brokers ip:port pairs (e.g."+
```

Comment:
We're planning to change some of this text soon, but at the moment this should still be "broker", which is a bit more obvious if we drop the ip:port descriptive text: `broker pairs` vs `brokers pairs`.